packages/plugin-lighthouse/package.json

```diff
@@ -5,6 +5,8 @@
   "dependencies": {
     "@code-pushup/models": "*",
     "lighthouse": "^11.0.0",
-    "@code-pushup/utils": "*"
+    "@code-pushup/utils": "*",
+    "lighthouse-logger": "2.0.1",
```

Comment:
It is possible but might be quite tricky to get rid of this dependency so I keep it in for now.